neurodsp/tests/sim/test_aperiodic.py

```diff
@@ -33,6 +35,26 @@ def test_sim_powerlaw():
     sig = sim_powerlaw(N_SECONDS, FS, f_range=(2, None))
     check_sim_output(sig)
 
+def test_sim_fgn():
+
+    # Simulate white noise.
+    sig = sim_fgn(N_SECONDS, FS)
+    check_sim_output(sig)
+
+    # Check the accuracy of the mean and standard deviation.
```

Comment:
It seems to me that these check the performance of the normalization, and not the simulation per se? If this is the case (if I'm not not missing something about it), I don't think we want these checks, as tests should be modular, directly testing only the function under question, and this tests something else (alternately stated: if this test fails, it's not `sim_fgn` that is broken).

Reply:
This is a good point. Perhaps it's best to remove the accuracy test. Effectively I was just using a sanity check that the first two moments of the empirical signal agreed with that of a standard gaussian.

Reply:
Or perhaps I avoid using normalization and check the first four empirical moments.